api-catalog-services/src/test/java/com/ca/mfaas/apicatalog/services/cached/CachedProductFamilyTest.java

```diff
@@ -68,7 +67,6 @@ public void testRetrievalOfRecentlyUpdatedContainers() {
     }
 
     @Test
-    @Ignore
```

Comment:
Do we want to unignore tests which contain Wait?

Reply:
Sonar test environment rules require these changes. Actually, I removed because I thought that it's already fixed in this PR.
https://github.com/zowe/api-layer/pull/303